src/DotNetWorker/Context/GrpcValueProvider.cs

```diff
@@ -1,42 +1,60 @@
-// Copyright (c) .NET Foundation. All rights reserved.
+﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using Google.Protobuf.Collections;
 using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
 
 namespace Microsoft.Azure.Functions.Worker.Context
 {
     internal class GrpcValueProvider : IValueProvider
     {
-        private readonly IDictionary<string, ParameterBinding> _inputData;
+        private readonly IDictionary<string, TypedData> _inputData;
+        private readonly IDictionary<string, TypedData> _triggerMetadata;
 
-        public GrpcValueProvider(IEnumerable<ParameterBinding> inputData)
+        public GrpcValueProvider(IEnumerable<ParameterBinding> inputData, MapField<string, TypedData> triggerMetadata)
         {
-            if (inputData is null)
+            if (inputData == null)
```

Comment:
Did something prompt this change? Curious, as I have been using `is` around for the null check.

Reply:
no, is is fine... i just saw an is and an == right next to each other.... I believe Fabio meant that we can start using is.